src/main.cpp

```diff
@@ -155,14 +155,17 @@ int main(int argc, char *argv[])
 #ifdef IS_QTCONTROLS_QT
     tr_path = ":/i18n";
 #endif
+#ifdef IS_UBPORTS
+    tr_path = "./translations";

```

Comment:
This large patch in C++ is essentially because of this line. Another comment will come as a general one